schemas/gltf-asset.schema.json

```diff
@@ -13,6 +13,14 @@
         "type": "Asset type should be a string."
       }
     },
+    "bodyType": {
+      "description": "The bodyType of asset.",
+      "type": "string",
+      "default": "custom",
+      "errorMessage": {
+        "type": "Bodytype should be a string."
```

Comment:
Please add a space between Body and type, the it matches with assetType error message.